.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
You still need to pass a toolchain root (with -sysroot)... which means you actually need a special toolchain, no?